seacrowd/sea_datasets/gatitos/gatitos.py

```diff
@@ -0,0 +1,158 @@
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+The GATITOS (Google's Additional Translations Into Tail-languages: Often Short) dataset is a high-quality, multi-way parallel dataset of tokens and short phrases.
+This dataset consists in 4,000 English segments (4,500 tokens) that have been translated into each of 173 languages, 170 of which are low-resource, 23 are spoken in Southeast Asia.
+This dataset contains primarily short segments: 93% single tokens, and only 23 sentences (0.6%) have over 5 tokens.
+As such it is best thought of as a multilingual lexicon, rather than a parallel training corpus.
+The source text is frequent words in the English Language, along with some common phrases and short sentences.
+Care has been taken to ensure that they include good coverage of numbers, months, days of the week, swadesh words, and names of the languages themselves (including the endonym).
+"""
+from pathlib import Path
+from typing import Dict, List, Tuple
+
+import datasets
+
+from seacrowd.utils import schemas
+from seacrowd.utils.configs import SEACrowdConfig
+from seacrowd.utils.constants import Licenses, Tasks
+
+_CITATION = """\
+@misc{jones2023bilex,
+      title={Bilex Rx: Lexical Data Augmentation for Massively Multilingual Machine Translation},
+      author={Alex Jones and Isaac Caswell and Ishank Saxena and Orhan Firat},
+      year={2023},
+      eprint={2303.15265},
+      archivePrefix={arXiv},
+      primaryClass={cs.CL}
+    }
+}
+"""
+
+_DATASETNAME = "gatitos"
+
+_DESCRIPTION = """\
+The GATITOS (Google's Additional Translations Into Tail-languages: Often Short) dataset is a high-quality, multi-way parallel dataset of tokens and short phrases.
+This dataset consists in 4,000 English segments (4,500 tokens) that have been translated into each of 173 languages, 170 of which are low-resource, 23 are spoken in Southeast Asia.
+This dataset contains primarily short segments: 93% single tokens, and only 23 sentences (0.6%) have over 5 tokens.
+As such it is best thought of as a multilingual lexicon, rather than a parallel training corpus.
+The source text is frequent words in the English Language, along with some common phrases and short sentences.
+Care has been taken to ensure that they include good coverage of numbers, months, days of the week, swadesh words, and names of the languages themselves (including the endonym).
+"""
+
+_HOMEPAGE = "https://github.com/google-research/url-nlp/blob/main/gatitos/README.md"
+
+_LANGUAGES = ["ace", "ban", "bbc", "bew", "bjn", "bts", "btx", "bug", "cnh", "hil", "iba", "ilo", "kac", "lus", "mad", "mak", "meo", "min", "pag", "pam", "shn", "tet", "war"]
+
+_LICENSE = Licenses.CC_BY_4_0.value
+
+_LOCAL = False
+_URL_TEMPLATE = "https://raw.githubusercontent.com/google-research/url-nlp/main/gatitos/{src}_{tgt}.tsv"
```

Comment:
can you rename it to `_URLS` so the variable convention is the same with the others?